docs/workflows/api-token.md

```diff
@@ -95,6 +95,11 @@ In the Beyond Identity Admin Console under the "Apps" tab, select the "Beyond Id
 
 ## Revoking Access Tokens
 
+Depending on what information is available to you, you may choose to use one of the ways to revoke an access token.
+It's recommended to use RFC7009 revoke endpoint if you can.
+
+### RFC7009 revoke endpoint
+
 In order to revoke an access token, you must be passed authentication in the form of either Bearer or Basic.
```

Comment:
Does this option also "revoke a token via management API"?

Reply:
Yes, it does use management API under the hood. The outcome of these two endpoints is the same, it's just a different API.

Reply:
Down below for `### Revoking token by token_id` says:

"In order to revoke a token via management API, you must pass a Bearer
 authentication containing a management API token with the tokens:delete
 scope." 
 
Do you think we should should mention the management API here too? Or if not, I'm trying to figure what you mean by the management API below? Is the /token part of management API and /revoke part of something else?

Reply:
Great call! I think I made this up, but in my mind, our APIs are divided to "management API" and "auth API" (`api-` vs `auth-` prefix).

In this context, "revoke a token via management API" simply means "revoke a token using something else than the RFC-7009 endpoint". Maybe it should be written as "In order to revoke a token by token_id" for consistency with section header. Would that make more sense?

Reply:
Ah ok, yea that might make more sense